src/main/java/edu/rosehulman/quota/controller/SetConfigController.java

```diff
@@ -33,19 +32,13 @@ public Object handle(Request request, Response response) throws Exception {
     // TODO: All of the database calls assume this is the first time the config has been uploaded
 
     JsonObject partnerJsonObject = new JsonParser().parse(body).getAsJsonObject();
-    String partnerId = partnerJsonObject.get("partnerId").getAsString();
-    String apiKey = partnerJsonObject.get("apiKey").getAsString();
-
-    Partner partner = new Partner();
-    partner.setPartnerId(partnerId);
-    partner.setApiKey(apiKey);
 
-    addPartnerToDatabase(partner);
+    String partnerId = partnerJsonObject.get("partnerId").getAsString();
 
     JsonArray productsJsonArray = partnerJsonObject.getAsJsonArray("products");
     productsJsonArray.iterator().forEachRemaining(productJsonElement -> {
       JsonObject productJsonObject = productJsonElement.getAsJsonObject();
-      String productId = productJsonObject.get("id").getAsString();
+      String productId = productJsonObject.get("productId").getAsString();
```

Comment:
Are all the Id changes from the new format of the config?

Reply:
This was a bug, `id` is *shared's* id for the product, not the actual product id. We need to use `productId` to get the actual id.